mkdocs.yml

```diff
@@ -1,77 +1,5 @@
 ---
-site_name: Datatrail CLI
-site_author: Tarik Zegmott
-site_description: "Datatrail CLI"
-repo_name: chimefrb/datatrail-cli
-repo_url: "https://github.com/chimefrb/datatrail-cli"
-copyright: Copyright &copy; 2016 - 2023 CHIME/FRB Collaboration
-
-theme:
-  name: material
-  palette:
-    scheme: slate
-    primary: deep purple
-    accent: yellow
-  font:
-    text: Roboto
-    code: Roboto Mono
-  language: en
-  features:
-    - navigation.expand
-    - navigation.top
-    - navigation.instant
-    - navigation.tabs
-    - navigation.tabs.sticky
-    - content.code.annotate
-
-plugins:
-  - search
-  - mkdocstrings
-  - termynal
-
-# Extensions
-markdown_extensions:
-  - admonition
-  - abbr
-  - attr_list
-  - def_list
-  - footnotes
-  - mkdocs-click
-  - md_in_html
-  - meta
-  - toc:
-      permalink: true
-  - pymdownx.arithmatex:
-      generic: true
-  - pymdownx.betterem:
-      smart_enable: all
-  - pymdownx.caret
-  - pymdownx.critic
-  - pymdownx.details
-  - pymdownx.emoji
-  - pymdownx.highlight:
-      use_pygments: true
-      linenums_style: pymdownx.inline
-  - pymdownx.inlinehilite
-  - pymdownx.keys
-  - pymdownx.magiclink:
-      repo_url_shorthand: true
-      user: squidfunk
-      repo: mkdocs-material
-  - pymdownx.mark
-  - pymdownx.smartsymbols
-  - pymdownx.snippets:
-      check_paths: true
-  - pymdownx.superfences:
-      custom_fences:
-        - name: mermaid
-          class: mermaid
-          format: "!!python/name:pymdownx.superfences.fence_code_format"
-  - pymdownx.tabbed:
-      alternate_style: true
-  - pymdownx.tasklist:
-      custom_checkbox: true
-  - pymdownx.tilde
+site_name: datatrail-cli
 
 nav:
   - Home:
```

Comment:
The rest of the `nav` seems to be missing? Currently, the Handbook generates it as follows:

``` yaml
nav:
  - Home:
      - Datatrail CLI: index.md
  - User Guide:
      - Welcome: user_guide.md
      - Install: install.md
      - Initialise: initialising.md
      - Commands:
          - clear: clear.md
          - list: list.md
          - ps: ps.md
          - pull: pull.md
          - scout: scout.md
  - Command Line Interface:
      - Commands: commands.md
      - Reference: cli.md
 ```

Reply:
Also apologies for the late review, I totally missed this.